custom/icds_reports/ucr/data_sources/person_cases_v3.json

```diff
@@ -157,6 +157,12 @@
         "type": "raw",
         "property_name": "dob"
       },
+      {
```

Comment:
are you also planning to rebuild the existing cases in this UCR?

Reply:
As of this time it's been mentioned that there's no backfill required for this so not yet. It could always be done at some future date.